src/fairseq2/metrics/accuracy.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from __future__ import annotations
+
+from typing import Iterable, Optional, Union, final
+
+import torch
+from torch import Tensor
+from torcheval.metrics import Metric
+from typing_extensions import Self
+
+from fairseq2.typing import Device, override
+
+
+@final
+class AccuracyMetric(Metric[Tensor]):
```

Comment:
Any chance of using one of the implementations in https://github.com/pytorch/torcheval/blob/main/torcheval/metrics/classification/accuracy.py instead of this implementation?

Reply:
Done! Thank you for the pointer, removed this unnecessary Metric.